spec/features/index_user_spec.rb

```diff
@@ -0,0 +1,38 @@
+require "rails_helper"
+
+feature "user views show page", %{
+  As a user
+  I want to view a food truck's information
+  so that I can learn more about it
+} do
+ # Acceptance Criteria
+ # * If a user goes to the show page, they see a list of food trucks.
+ scenario "user navigates to a food truck's show page" do
+   sample_food_truck_1 = FoodTruck.create(
+     name: "Chicken & Rice Guys",
+     description: "Food truck that specializes in chicken and lamb shawarma.",
+     avg_rating: 4,
+     location: "Harvard Square"
+   )
+
+   visit "/food_trucks/#{sample_food_truck_1.id}"
+ end
+
+ scenario "user views show page of a food truck" do
+   sample_food_truck_1 = FoodTruck.create(
+     name: "Chicken & Rice Guys",
+     description: "Food truck that specializes in chicken and lamb shawarma.",
+     avg_rating: 4,
+     location: "Harvard Square"
+   )
+
+   visit food_trucks_path
+   click_link(sample_food_truck_1.name)
+   visit food_truck_path(sample_food_truck_1.id)
```

Comment:
Once you've fixed that, leave a blank line between this portion and the expect statements.